docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,611 @@
+
+The benchmark configuration file may be provided in a yaml or json format: here we will use a yaml format. Please note that yaml files are case sensitive and all labels are in lowercase. 
+
+The benchmark configuration file has a single required stanza:
+
+``` bash
+test:
+```
+
+### Create A Template Benchmark Configuration File
+
+Under the **benchmarks** folder create a file called **myAssetBenchmark.yaml** with the following content:
+
+```yaml
+test:
+    name: asset-transfer-basic-benchmark
+    description: A test benchmark
+    workers:
+    rounds:
+```
+
+__test__: The root level block that contains the benchmark test information.
+
+__name__: The name of the test, in this case "asset-transfer-basic-benchmark".
+
+__descriptions__: A description for the benchmark, in this case "A test benchmark".
+
+__workers__: A set of keys used to define the number of workers (separate worker client instances) used in the subsequent benchmark.
+
+__rounds__: An array of distinct test rounds that will be progressed sequentially. Rounds may be used to benchmark different smart contract methods, or the same method in a different manner. 
+
+### Populating the Template File
+We will now populate the template file to specify the number of workers and the test round that uses the workload module we have created.
+
+#### Populate Workers
+We will be using two separate workers, this is accomplished through the workers specification:
+
+``` yaml
+  type: local
+  number: 2
+```
+
+#### Populate Rounds
+Each `round` block contains the following: 
+- `label` - the unique header label to use for the round.
+- `description` - a description of the round being run.
+- `txDuration` - the specification of the test duration, in seconds
+- `rateControl` - a rate control type, with options. 
+- `workload` - the workload module to use, with arguments to pass to the module. All arguments passed are available as `roundArguments` within the workload module.
+
+We will specify a benchmark round labeled `queryAsset`, with the description `Query asset benchmark`, to run for a 30s duration, using a `fixed-load` rate controller aiming to maintain a constant transaction pressure of 2. Additionally we will be providing a workload through specification of our `queryAssetBenchmarks.js` workload file, which we will pass the arguments `{assets: 10, contractId: asset-transfer-basic}`.
+
+The above is accomplished through the round specification:
+
+``` yaml
+    - label: queryAsset
+      description: Query asset benchmark
+      txDuration: 30
+      rateControl: 
+        type: fixed-load
+        opts:
+          transactionLoad: 2
+      workload:
+        module: workload/queryAssetBenchmarks.js
+        arguments:
+          assets: 10
+          contractId: asset-transfer-basic
```

Comment:
See comment above about the contract id, but I think this should be called `basic`